LICENSE

```diff
@@ -0,0 +1,21 @@
+The MIT License
+
+Copyright (c) 2010-2016 Google, Inc. http://angularjs.org
```

Comment:
Im pretty sure Google doesn't own this repo => "Copyright (c) 2010-2016 Google, Inc. http://angularjs.org"